_ml-commons-plugin/api/model-apis/register-model.md

```diff
@@ -258,17 +258,20 @@ Guardrails are safety measures for large language models (LLMs). They provide a
-
-#### Example request: Externally hosted model with guardrails
+Field | Data type | Description                                                                                                                                                                                                           
+:---  |:----------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+`type` | String    | The guardrail type. Currently, we have `local_regex` and `model` types.                                                                                                                                               
+`input_guardrail`| Object    | The guardrail for the model input.                                                                                                                                                                                    |
+`output_guardrail`| Object    | The guardrail for the model output.                                                                                                                                                                                   |
+`stop_words`| Object    | The list of indexes containing stopwords used for the model input/output validation. If the model prompt/response contains a stopword contained in any of the indexes, the predict request on this model is rejected. |
+`index_name`| Object    | The name of the index storing the stopwords.                                                                                                                                                                          |
+`source_fields`| Object    | The name of the field storing the stopwords.                                                                                                                                                                          |
+`regex`| Object    | A regular expression used for input/output validation. If the model prompt/response matches the regular expression, the predict request on this model is rejected.                                                    |
+`model_id`| String    | The guardrails model to validate user input and LLM output.                                                                                                                                                           |
+`response_filter`| String    | The dotpath used to get the result from guardrails model response.                                                                                                                                                    |
+`response_accept`| String    | The regex expression to check the guradrails model result.                                                                                                                                                            |
```

Comment:
This name `response_accept` may not be inherently descriptive of its function. The parameter is used to specify a regex pattern to validate the response, you may want to consider a more descriptive name such as `response_validation_regex` or `response_pattern`

Reply:
Makes sense. It needs code change, I can have it in my next pr.

Reply:
@jngz-es , when you have PR merged, please replace `response_accept` in this PR

Reply:
https://github.com/opensearch-project/ml-commons/pull/2571